docs/src/modules/components/PickersRenderingInstructions.js

```diff
@@ -21,7 +21,16 @@ export default function PickersRenderingInstructions() {
 
   const commandLines = [
     `import { LocalizationProvider } from '${componentPackage}';`,
+    ...(libraryUsed === 'date-fns'
+      ? ['// If you are using date-fns v2.x, please import `AdapterDateFns`']
+      : []),
     `import { ${adapterName} } from '${componentPackage}/${adapterName}'`,
+    ...(libraryUsed === 'date-fns'
+      ? [
+          '// If you are using date-fns v3.x, please import `AdapterDateFnsV3`',
+          `import { AdapterDateFnsV3 } from '${componentPackage}/AdapterDateFnsV3'`,
```

Comment:
this should be still `{ AdapterDateFns }`

Reply:
@LukasTy I would have prefered to stay consistent with import path = name of the main element exported :grimacing:

Reply:
🙈 Fair point.
My idea was to reduce the amount of changes that need to be done. 🤔 
I believe that in the future we will only export a single adapter and that migration can be codemoded pretty easily.

Reply:
I guess we can fix the doc now, doing a BC is not worth it :+1:

Reply:
Fixed in https://github.com/mui/mui-x/pull/12036